gatling/user-files/simulations/HttpBulkImport.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2011-2018 GatlingCorp (https://gatling.io)
```

Comment:
I don't think that this is the correct license header (I don't believe that we need one here).

If that remark trigger a change, then that change should be reported to all other similar headers.